### PR TITLE
Layer conf

### PR DESCRIPTION
**Pull request by flynnplatt**
Some patches that came up from walkthrough.

These changes reduce the amount of items needed in the config to get a working wmts layer

### CODE REVIEW

File: src/_core/reducers/models/map.js [hunk 1]
```diff
@@ -85,13 +85,13 @@ export const layerModel = Immutable.fromJS({
...
         url: "",
         layer: "",
-        format: "",
-        requestEncoding: "",
-        matrixSet: "",
-        projection: "",
-        extents: [],
+        format: "image/png",
+        requestEncoding: "KVP",
+        matrixSet: undefined,
```
Comment (AaronPlave) [inline src/_core/reducers/models/map.js]: What's the reason for `""` -> `undefined`

File: src/_core/utils/TileHandler.js [hunk 2]
```diff
@@ -134,7 +134,7 @@ export default class TileHandler {
     static _defaultKVPUrl(options) {
         let layer = options.layer;
         let url = this.mapUtil.buildTileUrl({
-            layerId: layer.get("id"),
+            layerId: layer.getIn(["wmtsOptions", "layer"]),
```
Comment (AaronPlave) [inline src/_core/utils/TileHandler.js]: Forget, what's the difference between these two ids?

File: src/_core/reducers/models/map.js (hunk 1 above)
Comment (flynnplatt) [reply src/_core/reducers/models/map.js]: the empty string is valid input for openlayers but doesn't do anything and just results in silent failure. Openlayers should complain about `undefined`

File: src/_core/utils/TileHandler.js (hunk 2 above)
Comment (flynnplatt) [reply src/_core/utils/TileHandler.js]: `layer.get("id")` is the unique id within the application, `layer.getIn(["wmtsOptions", "layer"])` is the layer id that should be used in the tile requests

File: src/_core/reducers/models/map.js (hunk 1 above)
Comment (AaronPlave) [reply src/_core/reducers/models/map.js]: Cool.

File: src/_core/utils/TileHandler.js (hunk 2 above)
Comment (AaronPlave) [reply src/_core/utils/TileHandler.js]: Got it. Feel like these are the same a lot of the time. Eventually we need to add comments/docs to all of our models to explain these things

**Comment by AaronPlave** [comment]
Ready to merge?

**Comment by flynnplatt** [comment]
ready for me

**Pull request merged**